examples/TLS_DenseSkOp.cpp

```diff
@@ -0,0 +1,164 @@
+#include <blas.hh>
+#include <RandBLAS.hh>
+#include <lapack.hh>
+
+#include <omp.h>
+#include <stdio.h>
+#include <unistd.h>
+#include <iostream>
+#include <math.h>
+#include <time.h>
+#include <stdlib.h>
+#include <chrono>
+
+using std::chrono::high_resolution_clock;
+using std::chrono::duration_cast;
+using std::chrono::duration;
+using std::chrono::milliseconds;
+
+
+void init_noisy_data(int64_t m, int64_t n, int64_t d, double* AB){
+    double target_x[n*d];
+    double eps[m*d];
+    for (int i = 0; i < n; i++) {  
+        target_x[i] = 1;   // Target X is the vector of 1's
+    }
+
+    RandBLAS::DenseDist Dist_A(m,n); 
+    RandBLAS::DenseDist Dist_eps(m,d); 
+    auto state = RandBLAS::RNGState(0);
+    auto state1 = RandBLAS::RNGState(1);
+
+    RandBLAS::fill_dense<double>(Dist_A, AB, state);  //Fill A to be a random gaussian
+    RandBLAS::fill_dense<double>(Dist_eps, eps, state1);  //Fill A to be a random gaussian
+
+    blas::gemm(blas::Layout::ColMajor, blas::Op::NoTrans, blas::Op::NoTrans, m, d, n, 1, AB, m, target_x, n, 0, &AB[m*n], m);
+
+    for (int i = 0; i < m*d; i++){
+        AB[m*n + i] += eps[i];   // Add Gaussian Noise to right hand side
+    }
+}
+
+int main(int argc, char* argv[]){
+    // Goal: Solve total least squares problem ||AX - B|| 
```

Comment:
@kaiwenhe7 This is a little misleading. Please write a brief description of the total least squares problem. Please also explain how to call this executable.